### PR TITLE
Add missing /o/ in resource context for org level APIs

### PR DESCRIPTION
**Pull request by dewniMW**
Issue: https://github.com/wso2/product-is/issues/18603

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2 [hunk 1]
```diff
@@ -832,12 +832,12 @@
         </Resource>
 
         <!-- [Organization] Admin Application Management API -->
-        <Resource context="(.*)/api/server/v1/applications/(.*)/owner(.*)" secured="true" http-method="PUT">
-            <Scopes>internal_organization_admin</Scopes>
+        <Resource context="(.*)/o/api/server/v1/applications/(.*)/owner(.*)" secured="true" http-method="PUT">
```
Comment (sadilchamishka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2:835]: Checked this file. There was no tenant level entry for 
`(.*)/api/server/v1/applications/(.*)/owner(.*)` after refactoring as here. Don't we need tenant API for that?

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2 [hunk 2]
```diff
@@ -832,12 +832,12 @@
...
+            <Scopes>internal_org_organization_admin</Scopes>
         </Resource>
         <!-- Should this go to applications mgt API -->
-        <Resource context="(.*)/api/server/v1/organizations/(.*)/share" secured="true" http-method="POST">
-            <Scopes>internal_application_mgt_update</Scopes>
+        <Resource context="(.*)/o/api/server/v1/organizations/(.*)/share" secured="true" http-method="POST">
```
Comment (sadilchamishka) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2:839]: Application sharing is not organization perspective API. The previous way looks correct.

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2 (hunk 1 above)
Comment (dewniMW) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2]: fixed

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2 (hunk 2 above)
Comment (dewniMW) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2]: fixed

**Review by sadilchamishka — APPROVED** [review]

**Pull request merged**